mflowgen/Tile_MemCore/construct.py

```diff
@@ -470,29 +460,22 @@ def construct():
   pin_idx = order.index( 'pin-assignments.tcl' ) # find pin-assignments.tcl
   order.insert( pin_idx + 1, 'edge-blockages.tcl' ) # add here
   init.update_params( { 'order': order } )
-
+  
   # Adding new input for genlibdb node to run
 
-  if adk_name == 'gf12-adk':
-    # gf12 uses synopsys-ptpx for genlib (default is cadence-genus)
+  # No longer conditional---amber and onyx now use same genlib mechanism
+  if True:
```

Comment:
It would be better to get rid of this comment and if statement

Reply:
done.